config/manifests/bases/saas-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,298 @@
+        name: echo-api
+        version: v1
+      - kind: Service
+        name: echo-api
+        version: v1
+      version: v1alpha1
+    - description: Defines the desired state of a 3scale SaaS MappingService instance
+      displayName: Mapping Service
+      kind: MappingService
+      name: mappingservices.saas.3scale.net
+      resources:
+      - kind: Deployment
+        name: mapping-service
+        version: v1
+      - kind: Service
+        name: mapping-service
+        version: v1
+      - kind: SecretDefinition
+        name: mapping-service-system-master-access-token
+        version: v1alpha1
+      - kind: GrafanaDashboard
+        name: threescale-mapping-service
+        version: v1alpha1
+      version: v1alpha1
+    - description: Defines the desired state of a 3scale SaaS System instance
+      displayName: System
+      kind: System
+      name: systems.saas.3scale.net
+      resources:
+      - kind: ConfigMap
+        name: system-environment
+        version: v1
+      - kind: Deployment
+        name: system-app
+        version: v1
+      - kind: Deployment
+        name: system-sidekiq
+        version: v1
+      - kind: Service
+        name: system
+        version: v1beta1
+      - kind: Ingress
+        name: multitenant-system
+        version: v1
+      - kind: StatefulSet
+        name: system-sphinx
+        version: v1
+      - kind: Service
+        name: system-sphinx
+        version: v1
+      - kind: ConfigMap
+        name: system-sphinx
+        version: v1
+      - kind: SecretDefinition
+        name: system-app
+        version: v1alpha1
+      - kind: SecretDefinition
+        name: system-backend-listener
+        version: v1alpha1
+      - kind: SecretDefinition
+        name: system-config
+        version: v1alpha1
+      - kind: SecretDefinition
+        name: system-database
+        version: v1alpha1
+      - kind: SecretDefinition
+        name: system-events-hook
+        version: v1alpha1
+      - kind: SecretDefinition
+        name: system-master-apicast
+        version: v1alpha1
+      - kind: SecretDefinition
+        name: system-memcached
+        version: v1alpha1
+      - kind: SecretDefinition
+        name: system-multitenant-assets-s3
+        version: v1alpha1
+      - kind: SecretDefinition
+        name: system-recaptcha
+        version: v1alpha1
+      - kind: SecretDefinition
+        name: system-redis
+        version: v1alpha1
+      - kind: SecretDefinition
+        name: system-seed
+        version: v1alpha1
+      - kind: SecretDefinition
+        name: system-smtp
+        version: v1alpha1
+      - kind: GrafanaDashboard
+        name: threescale-system
+        version: v1alpha1
+      version: v1alpha1
+    - description: Defines the desired state of a 3scale SaaS Zync instance
+      displayName: Zync
+      kind: Zync
+      name: zyncs.saas.3scale.net
+      resources:
+      - kind: Deployment
+        name: zync
+        version: ""
```

Comment:
Why there is no `v1` version here? it is autogenerated or manual?

Reply:
Is manual, fixed in https://github.com/3scale/saas-operator/pull/52/commits/605387a4196e38742d1dffdb481facea4a4e56b7